src/appleseed/renderer/kernel/lighting/lighttracing/lighttracingsamplegenerator.cpp

```diff
@@ -862,6 +865,8 @@ namespace
                 ~0, // max diffuse bounces
                 ~0, // max glossy bounces
                 ~0, // max specular bounces
+                ~0, // max volume bounces
+                false,
```

Comment:
@dictoon I wanted to point out this change - I had to add `max volume bounces` which means it didn't have enough arguments and `false` was converted to `size_t max_volume_bounces`. Is it possible it was a bug?